src/pranitaPJun20/StringExtractItemID.java

```diff
@@ -0,0 +1,55 @@
+/*/*Assignment 33 : 22nd July 2020
+
+	Extract ItemId from below input
+
+	String input = https://www.facebook.com/technocredits/itemId=12345/
+	String input = https://www.facebook.com/technocredits/empid=123&itemId=12345/
+	String input = https://www.facebook.com/technocredits/itemId=12345&empid=123/
+	String input = https://www.facebook.com/technocredits/empid=123/
+
+	Output : 
+	item id is 12345 
+	        OR
+	no item id present.
+	 * 
+	 */
+package pranitaPJun20;
+
+public class StringExtractItemID {
+
+	void extractItemId(String input) {
+		System.out.println("\nInput = " + input);
+		String number = "";
+		if (input.contains("itemId")) {
+			String subInput = input.substring(input.lastIndexOf("itemId="), input.lastIndexOf("/"));
+			if (subInput.contains("empid")) {
+				subInput = subInput.substring(0, subInput.lastIndexOf("&"));
+			}
+			String[] array = subInput.split("");
+			for (int index = 0; index < array.length; index++) {
+				try {
+					number += Integer.parseInt(array[index]);
+				} catch (NumberFormatException nfe) {
```

Comment:
Suggesting an alternative You can also write : 
try{
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

}catch(StringIndexOutOfBoundsException siob){
			System.out.println("Item ID -> Not present");
		}

This two line in try block and it will work. 

Second solution.
if (input.contains("itemId=")) {
			String str = input.split("itemId=")[1];
			String output = "";
			for (int index = 0; index < str.length(); index++) {
				if (Character.isDigit(str.charAt(index))) {
					output += str.charAt(index);
				} else {
					break;
				}
			}
			System.out.println("Item id is " + output);
		} else {
			System.out.println("No itemID present.");
		}